build/commands/scripts/mcp-setup.js

```diff
@@ -0,0 +1,415 @@
+    const bravePromptPath = path.join(BRAVE_CORE_ROOT, 'agents', 'prompts', 'brave_developer.md');
+    
+    try {
+      if (fs.existsSync(bravePromptPath)) {
+        const promptContent = fs.readFileSync(bravePromptPath, 'utf8');
+        // Remove the markdown header and format for .cursorrules
+        const rulesContent = promptContent.replace(/^# Brave Browser Development Assistant\n\n/, '');
+        fs.writeFileSync(cursorRulesPath, rulesContent);
+        log(`✅ Created Cursor rules: ${cursorRulesPath}`, 'success');
+      }
+    } catch (error) {
+      log(`Warning: Could not create .cursorrules: ${error.message}`, 'warning');
+    }
+  }
+  
+  return mcpConfig;
+}
+
+function updateGlobalConfig(projectConfig) {
+  if (process.argv.includes('--global')) {
+    log('Updating global Claude configuration...');
+    
+    ensureDirectoryExists(path.dirname(GLOBAL_CLAUDE_CONFIG_PATH));
+    
+    let globalConfig = { mcpServers: {} };
+    
+    // Read existing global config if it exists
+    if (fs.existsSync(GLOBAL_CLAUDE_CONFIG_PATH)) {
+      try {
+        const existing = fs.readFileSync(GLOBAL_CLAUDE_CONFIG_PATH, 'utf8');
+        globalConfig = JSON.parse(existing);
+      } catch (error) {
+        log(`Warning: Could not parse existing global config: ${error.message}`, 'warning');
+      }
+    }
+    
+    // Merge with project config
+    globalConfig.mcpServers = {
+      ...globalConfig.mcpServers,
+      ...projectConfig.mcpServers
+    };
+    
+    fs.writeFileSync(GLOBAL_CLAUDE_CONFIG_PATH, JSON.stringify(globalConfig, null, 2));
+    log(`Updated global Claude config: ${GLOBAL_CLAUDE_CONFIG_PATH}`, 'success');
+  }
+}
+
+function testMCPServer() {
+  if (process.argv.includes('--skip-test')) {
+    return;
+  }
+  
+  log('Testing MCP server...');
+  
+  const testScriptPath = path.join(BRAVE_CORE_ROOT, 'agents', 'mcp', 'test_server.py');
+  const pythonCmd = getPythonCommand();
+  
+  try {
+    execSync(`${pythonCmd} "${testScriptPath}"`, { 
+      stdio: 'inherit',
+      cwd: PROJECT_ROOT 
+    });
+    log('MCP server test completed', 'success');
+  } catch (error) {
+    log('MCP server test failed - but this may be normal for first-time setup', 'warning');
+  }
+}
+
+function startMCPServer() {
+  if (!process.argv.includes('--start')) {
+    return;
+  }
+  
+  log('Starting MCP server...');
+  
+  const pythonCmd = getPythonCommand();
+  
+  const serverProcess = spawn(pythonCmd, ['-m', 'brave.agents.mcp.source_access_server'], {
+    cwd: PROJECT_ROOT,
+    stdio: 'inherit',
+    env: {
+      ...process.env,
+      PYTHONPATH: path.join(PROJECT_ROOT, 'src')
+    }
+  });
+  
+  process.on('SIGINT', () => {
+    log('Stopping MCP server...');
+    serverProcess.kill('SIGINT');
+    process.exit(0);
+  });
+  
+  serverProcess.on('error', (error) => {
+    log(`Server error: ${error.message}`, 'error');
+  });
+  
+  serverProcess.on('close', (code) => {
+    log(`Server exited with code ${code}`);
+  });
+}
```

Comment:
this should just call to the separate script or get consolidated so that we don't repeat ourselves.